lzy/worker/src/main/java/ai/lzy/worker/env/EnvironmentFactory.java

```diff
@@ -42,6 +42,9 @@ public AuxEnvironment create(String fsRoot, LME.EnvSpec env) {
             var image = env.getDockerImage();
             LOG.info("Creating env with docker image {}", image);
 
+            var envVars = new HashMap<>(env.getEnvMap());
```

Comment:
this copy is redundant
you may add extra env after `.setEnvs()` call (on line 56)